packages/commonwealth/client/scripts/utils/downloadDataAsFile.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Save the content locally and use the `filename` as the suggested name for the
+ * file.
+ */
+export function downloadDataAsFile(content: string, filename: string) {
+  const blob = new Blob([content], { type: 'text/markdown' });
```

Comment:
Nit: we could add the content type as a param.

Reply:
Actually, that's a bug... I just didn't see that in the original. I'll add that now.